### ISSUE TITLE
Why is MultiOutDemandUGen hidden in Dneuromodule?

### ISSUE THREAD

**Issue by LFSaw** [supercollider/sc3-plugins#139]
***

**Comment by LFSaw** [comment]
@telephon, maybe you know? I think we should have it in the main distro...

**Comment by telephon** [comment]
OK, yes, would make sense. I just needed it. Does it look OK?

**Comment by LFSaw** [comment]
Just by glancing over it, it looks quite reasonable... is it supposed to work for arbitrary multi-out demand-rate UGens? I could use this e.g. for an updated [DetaBlockerBuf](https://github.com/supercollider/sc3-plugins/blob/master/source/BetablockerUGens/BetablockerUGens.cpp#L24) UGen…

**Comment by LFSaw** [comment]
Btw., the 'Dneuromodule' looks fun :) did you ever think of training it?

**Comment by telephon** [comment]
>is it supposed to work for arbitrary multi-out demand-rate UGens?

Yes, it is just a general interface which could be moved to common. We just have to somehow make sure we don't get duplicate class errors.

**Comment by telephon** [comment]
training `Dneuromodule`, well the idea behind it is less in training as such but in the complex dynamics. But Frank Pasmann has done a lot of work on them, for control of robots etc. 

The pattern version is more flexible though: https://github.com/telephon/Neuromodule
